lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -2166,6 +2166,12 @@ class DefaultAssetPickerBuilderDelegate
     );
   }
 
+  Future<void> presentLimitedAndReload() async {
+    await PhotoManager.presentLimited();
```

Comment:
The call will end immediately on iOS 14, making the rest of the code invalid.

Reply:
Updated: keep old logic for ios